src/Illuminate/Routing/Console/ControllerMakeCommand.php

```diff
@@ -268,6 +270,35 @@ protected function generateFormRequests($modelClass, $storeRequestClass, $update
         return [$storeRequestClass, $updateRequestClass];
     }
 
+    /**
+     * Build the return type replacement values.
+     *
+     * @param  array  $replace
+     * @param  string|null  $return
+     * @return array
+     */
+    protected function buildReturnTypeReplacements(array $replace, string|null $returnType)
```

Comment:
Two types in the docblock are already redundant, but `: array` should also be moved to a native typehint.